pylint/checkers/refactoring.py

```diff
@@ -174,6 +174,11 @@ class RefactoringChecker(checkers.BaseTokenChecker):
                   'state this as return None, and an explicit return statement '
                   'should be present at the end of the function (if reachable)'
                  ),
+        'R1711': ("Useless return at end of function or method",
+                  'useless-return',
+                  'Emitted when a single "return" or "return None" statement is found '
```

Comment:
Can you add another sentence saying why this can be removed? e.g because the function has an implicit return of None etc.